.gitignore

```diff
@@ -29,7 +29,6 @@ vendor/
 .DS_Store
 bin/
 tmp/
-crds/
```

Comment:
If we move this line, developers might apply the outdated manifests if the forget to `make generate` and use those manifests from this present crds folder, I am afraid.